nCompiler/R/compile_labelAbstractTypes.R

```diff
@@ -152,6 +155,9 @@ inLabelAbstractTypesEnv(
 inLabelAbstractTypesEnv(
     Assign <- 
         function(code, symTab, auxEnv, handlingInfo) {
+            logging <- loggingIsOn()
```

Comment:
I was thinking that rather than inserting code like this into every handler, we could put it in `compile_labelAbstractTypes`, right before calling the handler at `ans <- eval(call(handler ...))`.  That way one line of code could always log what handler is being called for what `code$name`.

Reply:
Thank you, that's much cleaner. Updated in d76965e.